src/site/content/en/blog/css-size-adjust/index.md

```diff
@@ -0,0 +1,395 @@
+---
+title: CSS `size-adjust` for `@font-face`
+subhead: As a web font loads, you can now adjust its scale, to normalize the document font sizes and prevent layout shift when switching between fonts
+authors:
+  - adamargyle
+description: As a web font loads, you can now adjust its scale to normalize the document font sizes and prevent layout shift when switching between fonts
+hero: image/vS06HQ1YTsbMKSFTIPl2iogUQP73/fL3dWSY7YBfagFP0ikMW.jpg
+alt: Well used letters from a letterpress, set into rows
+tags:
+  - blog
+  - css
+date: 2021-06-04
+updated: 2021-06-04
+---
+
+Consider the following
+[demo](https://codepen.io/argyleink/pen/8ace843980e2fd24bf15c8bd1bd072e4?editors=1100)
+where the `font-size` is a consistent `64px`, and the only difference between each of these headers is the `font-family`. The examples on the left have not been adjusted and
+have an inconsistent final size. The examples on the right use `size-adjust` to
+ensure `64px` is the consistent final size.
+
+<figure class="w-figure w-figure--fullbleed">
+  {% Video
+    src="video/vS06HQ1YTsbMKSFTIPl2iogUQP73/6xhatUhszn7M6pDFcVgV.mp4",
+    autoplay="true",
+    loop="true",
+    muted="true"
+  %}
+  <figcaption class="w-figure">
+   In this example Chrome DevTools CSS grid layout debug tools are used to show heights.
+  </figcaption>
+</figure>
+
+This post explores a [new CSS font-face
+descriptor](https://drafts.csswg.org/css-fonts-5/#size-adjust-desc) called
+`size-adjust`, available in Firefox 89 (behind a flag) and Chromium 92 (see
```

Comment:
I suggest listing Chromium first, since that's the shipping one.

Reply:
```suggestion
`size-adjust`, available in Chromium 92 and Firefox 89 (behind a flag); (see
```